### PR TITLE
feat: add documentation for advanced performance metrics

### PR DESCRIPTION
**Pull request by stefanjudis**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [ ] Docs
* [x] Learn
* [ ] Other

Adjusted the performance section to give more guidance on headless performance monitoring and testing. It's a rather lengthy PR so it'd be best to go over [the final result](https://checklyhq-cya2pb5mo-checkly.vercel.app/learn/headless/basics-performance/) and skim it there. 

@ragog I'd take this as a first step and iterate depending on your feedback. I think adding another section next to "Measuring page performance" would make total sense. Something like "Measure navigations across a session" or something.

### CODE REVIEW

File: site/content/learn/headless/basics-performance.md [hunk 1]
```diff
@@ -92,13 +108,102 @@ The Resource Timing API allows us to zoom in to single resources and get accurat
 {{< /tab >}}
 {{< /tabs >}}
 
+#### Paint Timing API (`first-paint` and `first-contentful-paint`)
+
+[The Paint Timing API](https://developer.mozilla.org/en-US/docs/Web/API/PerformancePaintTiming) provides information on the first paint and the first contentful paint. Access the entries via `performance.getEntriesByType('paint')` or `performance.getEntriesByName('first-contentful-paint')`.
+
+{{< tabs "3" >}}
```
Comment (ragog) [inline site/content/learn/headless/basics-performance.md:115]: @stefanjudis I am not sure I am sold on having the output of the API commented in the code. On one hand it saves space, but on the other I feel it takes away from clarity and tidiness. Do we want to just add a line below with something like "Here is the output:" followed by a code block with the output from the console.log?

**Review by ragog — DISMISSED** [review]
@stefanjudis I've added some comments I'd like you to take a look at, but all in all this LGTM.

File: site/content/learn/headless/basics-performance.md [hunk 2]
```diff
@@ -15,50 +15,66 @@ menu:
...
+Modern browsers support many APIs to gather web performance metrics and web vitals.
+
+#### Navigation and Resource Timing API
+
+The Navigation Timing and the Resource Timing performance APIs are W3C specifications. The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Performance/Navigation_and_resource_timings) define the scope of both:
 
 > Navigation timings are metrics measuring a browser's document navigation events. Resource timings are detailed network timing measurements regarding the loading of an application's resources. Both provide the same read-only properties, but navigation timing measures the main document's timings whereas the resource timing provides the times for all the assets or resources called in by that main document and the resources' requested resources.
 
-We can use the Navigation Timing API to retrieve timestamps of key events in the page load timeline.
+[The Navigation Timing API](https://developer.mozilla.org/en-US/docs/Web/API/Navigation_timing_API) allows us to retrieve timestamps of key events in the page load timeline. A Navigation Timing entry includes metrics such as the navigation response time, DOM ready and document load time.
```
[the remark this answers is not in the document]
Comment (stefanjudis) [reply site/content/learn/headless/basics-performance.md:77]: I'll remove DOM ContentLoaded and pick another one. :)

File: site/content/learn/headless/basics-performance.md (hunk 1 above)
Comment (stefanjudis) [reply site/content/learn/headless/basics-performance.md:115]: As discussed in Slack I'll introduce a new `details` block. 

image

image

**Comment by stefanjudis** [comment]
@ragog Here's the updated version. 

https://checklyhq-pj15e7oh8-checkly.vercel.app/learn/headless/basics-performance/

**Pull request merged**